File: pset1/mario-less/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int h;
    do
    {
        h = get_int("Height: ");
    }
    while (h < 1 || h > 8);

    for (int r = 0; r < h; r++)
    {
        for (int c = 0; c < h; c++)
        {
            if (r + c < h - 1)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }

        printf("\n");
    }
}

File: pset6/sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    h = get_int("Height: ")
    if 1 <= h <= 8:
        break

for r in range(h):
    for c in range(h):
        if r + c < h - 1:
            print(" ", end="")
        else:
            print("#", end="")

    print("  " + "#" * (r + 1))